index.php

```diff
@@ -596,26 +596,47 @@
         }
     }
     else {
-        $layout->renderBeforeContentHTML(
-            system: $system,
-            player: $player,
-            page_title: "Profile"
-        );
-
-        $system->printMessage();
-        if(!$player->global_message_viewed) {
-            $global_message = $system->fetchGlobalMessage();
-            $layout->renderGlobalMessage($system, $global_message);
-        }
+        if ($system->environment == System::ENVIRONMENT_DEV) {
+            $layout = $system->fetchLayoutByName("new_geisha");
+            $layout->renderBeforeContentHTML($system, $player ?? null, "Home", custom_page: true);
+            if (!$player->global_message_viewed) {
+                $global_message = $system->fetchGlobalMessage();
+                $layout->renderGlobalMessage($system, $global_message);
+            }
+            try {
+                require('./templates/home.php');
```

Comment:
Hmm - We intentionally have profile as the default page when user is logged in to avoid level holding by never visiting profile. Can we retain that behavior here?